src/Uno.UWP/UI/ColorHelper.cs

```diff
@@ -1,5 +1,9 @@
 ﻿using System;
 using System.ComponentModel;
+using Windows.UI;
+
+// Required for the WinUI auto upgrade;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5475653)